.vscode/extensions.json

```diff
@@ -0,0 +1,14 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
```

Comment:
I still don't think we should commit IDE configuration files.

Reply:
That's pretty convenient though, so I just open a solution/a folder and stuff works out of the box. It doesn't hurt to have it there.